_positions/hud-fellow-software-developer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: HUD Innovation Fellow - Software Developer
```

Comment:
About to merge for the sake of expediency, but for future consideration: where possible, it's nice to have more consistency among the filename, title and permalink.